### PR TITLE
Added default mappings

### PR DESCRIPTION
**Pull request by Kilobravo18**
This includes missing mappings for signals that are inside our default templates: default-ice-can11 and default-ice-can29
- intakePressure (kPa) (ranges: -0 to 255)
     VSS Def: https://github.com/DIMO-Network/model-garage/blob/main/pkg/schema/spec/vss_rel_4.2-DIMO-b2679e5c.csv#L1130
- shortTermFuel (%) (ranges: -100 to 99.21875)
     VSS Def: https://github.com/DIMO-Network/model-garage/blob/main/pkg/schema/spec/vss_rel_4.2-DIMO-b2679e5c.csv#L1125
- longTermFuel (%) (ranges: -100 to 99.21875)
     VSS Def: https://github.com/DIMO-Network/model-garage/blob/main/pkg/schema/spec/vss_rel_4.2-DIMO-b2679e5c.csv#L1126
- commandedEgr (%) ( 0 to 100) 
     VSS Def: https://github.com/DIMO-Network/model-garage/blob/main/pkg/schema/spec/vss_rel_4.2-DIMO-b2679e5c.csv#L1204
- distanceSinceDtcClear (km) (0 to 65535)
     VSS Def: https://github.com/DIMO-Network/model-garage/blob/main/pkg/schema/spec/vss_rel_4.2-DIMO-b2679e5c.csv#L1168
- warmupsSinceDtcClear (count) (0 to 255) 
     VSS Def: https://github.com/DIMO-Network/model-garage/blob/main/pkg/schema/spec/vss_rel_4.2-DIMO-b2679e5c.csv#L1208

### CODE REVIEW

**Comment by elffjs** [comment]
Are you sure about the units here? They’re supposed to be percent, often. Are you doing fractions or?

File: pkg/schema/spec/definitions.yaml [hunk 1]
```diff
@@ -295,3 +295,52 @@
...
+      isArray: false
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA
+- vspecName: Vehicle.OBD.WarmupsSinceDTCClear
+  conversions: 
+    - originalName: warmupsSinceDtcClear
+      originalType: float64
+      isArray: false
+  requiredPrivileges:
+    - VEHICLE_NON_LOCATION_DATA
```
Comment (elffjs) [inline pkg/schema/spec/definitions.yaml:346]: Please newlines at the ends of files.

**Comment by elffjs** [comment]
Like I would feel way more comfortable if it were clear to me that we understood the input and output units. I can’t find any `longFuelTrim`, for example, in Elastic so I can’t help you.

**Comment by elffjs** [comment]
Can we link this to an issue and/or have a description here?

**Comment by Kilobravo18** [comment]
> Like I would feel way more comfortable if it were clear to me that we understood the input and output units are. I can’t find any `longFuelTrim`, for example, in Elastic so I can’t help you.
**The unit is in percentage, inline with the csv we have in model garage. I loved in elastic at longFuelTrim1 and saw many returns all inline with the normal aspect of 10 to -10 ranges for the system. **
> Can we link this to an issue and/or have a description here?
**Not sure what you are asking here, can you give me some more context please?**

**Comment by elffjs** [comment]
You wrote `longFuelTrim` as the `originalName` [here](https://github.com/DIMO-Network/model-garage/pull/58/files#diff-e1b16970c7dec123f893840461e94344c192a5d269143757ab017737f679747eR307). Is that not right? 

I do see a `longTermFuelTrim1` in Elastic.

**Comment by elffjs** [comment]
Is it `longFuelTrim` in v2, maybe? And it gets mapped to `longTermFuelTrim1` in v1? I cannot keep track of what you guys are doing there.

**Comment by Kilobravo18** [comment]
> Is it longFuelTrim in v2, maybe? And it gets mapped to longTermFuelTrim1 in v1? I cannot keep track of what you guys are doing with these mappings.

**The answers you seek I can not give my liege, I am but a pawn in this great game of DIMO your confusion is shared**

**Comment by JamesReate** [comment]
> Is it `longFuelTrim` in v2, maybe? And it gets mapped to `longTermFuelTrim1` in v1? I cannot keep track of what you guys are doing with these mappings.

in ingest-autopi there is a mapping, so let's just change the name in the template
`        "longFuelTrim": "longTermFuelTrim1",`

**Comment by KevinJoiner** [comment]
Can you add a description to this PR with the following info
- What new signals are we trying to add?
- What testing have we done to verify that these signals are correctly decoded with the correct Units for example( percents are 0-100 for all MMYs)

**Comment by KevinJoiner** [comment]
I will fix test for you

**Review by KevinJoiner — APPROVED** [review]

**Pull request merged**